distributed-calculator/react-calculator/server.js

```diff
@@ -1,9 +1,13 @@
 const express = require('express');
 const path = require('path');
-const request = require('request');
+const bodyParser = require('body-parser');
+const axios = require('axios');
 
 const app = express();
 
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
```

Comment:
you can use `express.json()` here, so I am not sure bodyParser is required. Maybe try removing the dependency?